src/masternode.h

```diff
@@ -1,47 +1,47 @@
-
 // Copyright (c) 2014-2016 The Dash Core developers
 // Distributed under the MIT/X11 software license, see the accompanying
 // file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
 #ifndef MASTERNODE_H
 #define MASTERNODE_H
 
-#include "sync.h"
-#include "net.h"
 #include "key.h"
-#include "util.h"
-#include "base58.h"
 #include "main.h"
+#include "net.h"
 #include "timedata.h"
 
-#define MASTERNODE_MIN_MNP_SECONDS             (10*60)
-#define MASTERNODE_MIN_MNB_SECONDS             (5*60)
-#define MASTERNODE_PING_SECONDS                (5*60)
-#define MASTERNODE_EXPIRATION_SECONDS          (65*60)
-#define MASTERNODE_REMOVAL_SECONDS             (75*60)
-#define MASTERNODE_CHECK_SECONDS               5
-
-using namespace std;
-
 class CMasternode;
 class CMasternodeBroadcast;
 class CMasternodePing;
 
+static const int MASTERNODE_MIN_MNP_SECONDS       = 10 * 60;
+static const int MASTERNODE_MIN_MNB_SECONDS       =  5 * 60;
+static const int MASTERNODE_MIN_DSEG_SECONDS      = 10 * 60;
```

Comment:
Nope, MASTERNODE_PING_SECONDS is simply not used anywhere. It was replaced by MASTERNODE_MIN_MNP_SECONDS which stands for "minimum period of time after which new pings are allowed". MASTERNODE_MIN_DSEG_SECONDS is the same but for the dseg message (mn list sync).